README.md

```diff
@@ -0,0 +1,198 @@
+Classy Test
+===========
+
+Meteor package which provides a class-based wrapper around `tinytest`.
+
+It has the following features:
+
+ * Class-based test cases.
+ * Common setUp/tearDown methods for test cases, separated for server and client side.
+ * A single test can interleave client-side and server-side assertions.
+ * Support for asynchronous tests.
+ * Compatible with tinytest (all tests are actually registered via tinytest).
+
+Installation
+------------
+
+```
+meteor add peerlibrary:classy-test
+```
+
+Test cases
+----------
+
+Each test case is a class extending from `ClassyTestCase` as follows:
+
+```coffeescript
+class SimpleTestCase extends ClassyTestCase
+  testThatTrueIsTrue: =>
+    @assertTrue true, "True should be true."
+
+  testThatFalseIsFalse: =>
+    @assertFalse false, "False should be false."
+
+# Register the test case.
+ClassyTestCase.addTest new SimpleTestCase()
+```
+
+This simple test case definition will generate two tests via tinytest, the first will be called `SimpleTestCase - ThatTrueIsTrue` and the second one `SimpleTestCase - ThatFalseIsFalse`.
+
+Assertions
+----------
+
+Classy test assertions are named slightly differently than in tinytest, but are otherwise equivalent with some additional assertions provided by default. Because test cases are class-based, assertions are methods which may be called in test context. The list of assertions is as follows:
+
+ * `assertEqual(a, b, message)` asserts that `a` is equal to `b`.
```

Comment:
It is unclear which of the values should be the value you are testing, and which of the values should be the expected value. Or is this important? I do not see it in [unittest](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertEqual) documentation.

The reason why I am asking this is because when there is a string difference, for example, Meteor provides a nice diff. So having known what is marked with - and what with + is good. So that it is not inverted.